tests/safeds_stubgen/stubs_generator/__snapshots__/test_generate_stubs.ambr

```diff
@@ -0,0 +1,152 @@
+# serializer version: 1
+# name: test_class_attribute_creation
+  '''
+  package test_stub_generation.attribute_module
+  
+  class A {}
+  
+  class B {
+  	attr type_hint_public: Int
+  	attr no_type_hint_public
```

Comment:
1. Try to infer the type from the assigned value if mypy offers this functionality.
2. If this is not possible (in general or for a specific case, add a TODO)

Reply:
Created issue #34.